grin/src/Transformations/ExtendedSyntax/Util.hs

```diff
@@ -106,19 +107,19 @@ mapNameUseExp f = \case
 subst :: Ord a => Map a a -> a -> a
 subst env x = Map.findWithDefault x x env
 
--- variable reference substitution (non recursive)
```

Comment:
Why did you remove the none recursive comment? It is important to note that the function operates an a single AST constructor.

Reply:
Good point, I left those out.